{{cookiecutter.app_name}}/README.md

```diff
@@ -45,10 +45,10 @@ Run the following commands to bootstrap your environment if you are unable to ru
 ```bash
 cd {{cookiecutter.app_name}}
 {%- if cookiecutter.use_pipenv == "yes" %}
-pipenv install --dev
-pipenv shell
+python -m pipenv install --dev
+python -m pipenv shell
 {%- else %}
-pip install -r requirements/dev.txt
+python -m pip install -r requirements/dev.txt
```

Comment:
I'm not sure this one is necessary. `virtualenv` environments come with `pip` available by default, so there shouldn't be any issues with the wrong version of pip being used.